_includes/markdown/PHP.md

```diff
@@ -88,7 +88,46 @@ Here are a few key points:
 
 * A [taxonomy](https://wordpress.org/support/article/taxonomies/) is a tool that lets us group or classify posts.
 
-    [Post meta](https://wordpress.org/support/article/custom-fields/) lets us store unique information about specific posts. As such the way post meta is stored does not facilitate efficient post lookups. Generally, looking up posts by post meta should be avoided (sometimes it can't). If you have to use one, make sure that it's not the main query and that it's cached.
+*    [Post meta](https://wordpress.org/support/article/custom-fields/) lets us store unique information about specific posts. As such the way post meta is stored does not facilitate efficient post lookups. Generally, looking up posts by post meta should be avoided (sometimes it can't). If you have to use one, make sure that it's not the main query and that it's cached.
+
+* Always set default values for post_meta keys, especially if they are used to filter other queries. When writing queries filtering by post meta, never use the compare type ```NOT EXISTS```.
+
+    Queries that need to check if a given meta_key does not exist need to use `LEFT JOIN`, gathering a lot more data. In practice, this type of query destroys any other performance improvements because it requires the MySQL engine to run through all the rows in the selected set to make sure it catches any rows that meet the criteria.
+    
+    If all posts have a specific meta key, or if you can test for the presence of a given key, the engine can use a simple `INNER JOIN` and only touch the rows that already meet the criteria, saving a lot of time and processing on large tables.
```

Comment:
I'd recommend rewording `If all posts have a specific meta key, or if you can test for the presence of a given key` to `To search for the presence of a given key`. The 'if all posts have a specific meta key' doesn't make a ton of sense to me, but I could be missing something.